src/tiler/sql/user_reservable.sql

```diff
@@ -46,4 +62,7 @@
     ON (turf.group_id = trustedmapper.group_id
         AND trustedmapper.user_id = <%= user_id %>
         AND trustedmapper.is_approved)
+  LEFT JOIN core_user AS individual_mapper
+    ON (individual_mapper.id = <%= user_id %>
+        AND COALESCE(individual_mapper.individual_mapper, FALSE))
```

Comment:
In my testing, the available blockfaces on the reservables page updated immediately as I toggled the Individual Mapper flag on & off. I'll add the User table to the reservable cache buster for good measure.